components/net/tests/local_directory_listing.rs

```diff
@@ -0,0 +1,742 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#![cfg(not(target_os = "windows"))]
```

Comment:
Can we remove this one? I also don't know why it was added to the fetch test file, but unless there is a reason for it being here, we may as well remove it. If there is a reason, let's file a issue for it.

Reply:
That mysterious configuration line has now been removed.